### DISCUSSION TITLE
Rendering HTML dynamically

### DISCUSSION THREAD

**Discussion by AsafAgranat** [cmjoseph07/svelty-email#5] [General]
Hi, I'm trying to create an email template builder tool that creates templates dynamically. For this to work, I need to be able to render the HTML on demand (whenever a change happens in the tool). From what I gathered until now, the template render function happens on the server in the `load()` function. Whenever I try to call the svelty-email render within the app, I get an error that "template.render is not a function".
Is there a way to invoke the render on demand, outside of `load()`?

**Comment by simond07** [comment]
Hey,
I'm trying to use something similiar where the user should see the rendered email and get the html in a textarea.
I tried to use the render function on the client. This works if I load the page and then reload it. I still get the same "TypeError: template.render is not a function" error and am also unable to press a button.

Do you or anyone else have found a solution?
